src/Data/LoggerAbstract.php

```diff
@@ -103,6 +107,17 @@ public function setExcluded(Exclude $exclude)
         $this->exclude = $exclude;
     }
 
+    public function setAppVersionNumber($version)
+    {
+        $this->version = $version;
+        return $this;
+    }
+
+    public function getAppVersionNumber()
+    {
+        return $this->version instanceof Version ? $this->version->getVersionNumber() : $this->version;
```

Comment:
Did you mean to write something like:
`return $this->version instanceof Version ? $this->version->getVersionNumber() : null`

Reply:
No. $this->version can be null by itself so no need to do ": null". I'm just making the check to make sure the ->getVersionNumber method doesn't break